scripts/minici-presubmit-all-tests.sh

```diff
@@ -17,16 +17,16 @@ case $1 in
         export JOB_NAME="odo-minikube-pr-build"
         export SENDQUEUE="amqp.ci.queue.minikube.send"
         export SENDTOPIC="amqp.ci.topic.minikube.send"
-        export SETUP_SCRIPT="scripts/minikube-minishift-setup-env.sh minikube"
-        export RUN_SCRIPT="scripts/minikube-minishift-all-tests.sh minikube"
+        export SETUP_SCRIPT="scripts/minikube-minishift-setup-env.sh"
+        export RUN_SCRIPT="scripts/minikube-minishift-all-tests.sh"
         export TIMEOUT="2h15m"
         ;;
     minishift)
         export JOB_NAME="odo-minishift-pr-tests"
         export SENDQUEUE="amqp.ci.queue.minishift.send"
         export SENDTOPIC="amqp.ci.topic.minishift.send"
-        export SETUP_SCRIPT="scripts/minikube-minishift-setup-env.sh minishift"
-        export RUN_SCRIPT="scripts/minikube-minishift-all-tests.sh minishift"
+        export SETUP_SCRIPT="scripts/minikube-minishift-setup-env.sh"
+        export RUN_SCRIPT="scripts/minikube-minishift-all-tests.sh"
         export TIMEOUT="4h00m"
         ;;
     *)
```

Comment:
@prietyc123  Where do you set the value for CLUSTER?
Should need to export CLUSTER="minikuke" and CLUSTER="minishift" within each case block as applicable.
Also there is no need to remove parameters passed in lines 20-21 and 28-29 as this does not have an impact. If the firewall will be updated then later on we do not need to change those lines again.

Reply:
> Where do you set the value for CLUSTER?

I directly set env inside jenkins and calling it into scripts as it works for `${OCP4X_DOWNLOAD_URL}` https://github.com/openshift/odo/blob/master/scripts/run_all_tests.sh#L30 

> Also there is no need to remove parameters passed in lines 20-21 and 28-29 as this does not have an impact. If the firewall will be updated then later on we do not need to change those lines again.

I think we should change it because anyway we will be doing changes with the `case` once ci-firewall support parameter and I would prefer current approach of testing rather talking about future 🙂 IMO Passing the parameter might confuse people.

> [ssh:Fedora 32] go: found github.com/onsi/ginkgo/ginkgo in github.com/onsi/ginkgo v1.14.0
> [ssh:Fedora 32] ++ case ${CLUSTER} in
> [ssh:Fedora 32] ++ echo '<<< Need parameter set to minikube or minishift >>>'
> [ssh:Fedora 32] ++ exit 1

Need to look into it, TBH.

Reply:
As per our chat you are setting the parameter value in the Jenkins script, therefore approving. If it does not work using that approach then you will need to set it up in scripts/minici-presubmit-all-tests.sh within the case structure to reduce complexity.